### DISCUSSION TITLE
Can you set this up to have the newer conversation on the top instead of the bottom?

### DISCUSSION THREAD

**Discussion by bkarlan** [protz/thunderbird-conversations#1908] [Ideas]
I like your app but it's unusable to me if the newest message is at the bottom and I have to scroll down in a long conversation to view the newest message.  I don't see a way to make the newest message be on top.

**Comment by Standard8** [comment]
Please see this issue for the latest thoughts: https://github.com/thunderbird-conversations/thunderbird-conversations/issues/1900
